src/submission/exceptions/exceptions.ts

```diff
@@ -161,52 +189,56 @@ export const checkForProgramAndEntityExceptions = async ({
 
   // if there are submitted exceptions for this program, check if they match record values
   if (!(programException || entityException)) {
-    return { filteredErrors: schemaValidationErrors, normalizedRecord: record };
+    return { filteredErrors: schemaValidationErrors, normalizedRecord };
   }
 
   // check each validation error for a matching exception, and remove the validaiton if the value matches the exception value
   schemaValidationErrors.forEach(validationError => {
     const validationErrorFieldName = validationError.fieldName;
     const fieldValue = record[validationErrorFieldName];
 
-    // Field value is a type we cannot allow exceptions for, validate as normal
-    if (!isAllowedTypeForException(fieldValue)) {
-      filteredErrors.push(validationError);
-      return;
-    }
+    const fieldSchema = entitySchema?.fields.find(fieldFilter(validationErrorFieldName));
+    const valueType = fieldSchema?.valueType;
+
+    const validNumericExceptionValue =
+      (valueType === 'number' || valueType === 'integer') &&
+      isValidNumericExceptionType(fieldValue);
 
-    // missing required field, validate as normal, exceptions still require a submitted value
-    const isMissingRequiredField =
-      validationError.errorType ===
-      dictionaryEntities.SchemaValidationErrorTypes.MISSING_REQUIRED_FIELD;
+    let normalizedFieldValue: string | undefined = '';
+    let normalizedValue: string | string[] = '';
```

Comment:
Looks like we are checking the numeric field type correctly and then just setting the normalized value to `''` anyways.

This and the following section of code seem to be doing two things: 
1. validating the current fieldValue is of the correct type (`''` or `undefined` for numeric, single string for strings
2. normalizing the value (numeric: make it undefined, string: make it lower case, wrap in array if needed

However, on line 207 here we set the normalizedFieldValue to `''` when the numeric value needs to be `undefined` in order to have an allowedValue in the `validateFieldValueWithExceptions` function.

As I read it, this won't work. Is there a step I am missing or does this need to be updated slightly?

Reply:
See line 219 -- for numeric exceptions, we just use what was submitted
We initialize `normalizedFieldValue` as a string b/c a) exceptions were originally for string fields and b) only string fields need to be normalized
For numeric values, we aren't doing any normalization

Reply:
So connecting the dots with your other comments -- if it's a numeric field and the record value is `''` we should be normalizing to undefined. I'll update.